settings.py

```diff
@@ -128,19 +128,27 @@
 USE_TZ = True
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.10/howto/static-files/
+# https://docs.djangoproject.com/en/4.2/howto/static-files/
 STATIC_URL = '/static/'
 STATICFILES_DIRS = ['ui/dist']
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATICFILES_FINDERS = (
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
 )
+STORAGES = {
+    "staticfiles": {
+        "BACKEND": "whitenoise.storage.CompressedManifestStaticFilesStorage",
```

Comment:
Adds [whitenoise compression and caching support](https://whitenoise.readthedocs.io/en/stable/django.html#add-compression-and-caching-support) for static files.

It's possible there was a reason that we didn't enable this in the past

Reply:
well the reason might be because tests were failing with `Missing staticfiles manifest entry for 'admin/css/base.css'` with this enabled